gutenberg.php

```diff
@@ -28,6 +28,16 @@
 function the_gutenberg_project() {
 	global $post_type_object;
 	?>
+	<noscript>
+		<div class="error" style='position:absolute;top:32px;z-index:40'><p>
+		<?php
+			printf(
+				__( 'The Block Editor requires JavaScript, please try the <a href="%s">Classic Editor plugin</a>.', 'gutenberg' ),
```

Comment:
This will read better as two sentences.
"The Block Editor requires JavaScript. Please try the Classic Editor plugin."